pkg/manifest/manifest_test.go

```diff
@@ -720,6 +720,41 @@ func Test_include(t *testing.T) {
 				EnabledCapabilities: []configv1.ClusterVersionCapability{"cap1"},
 			},
 		},
+		{
+			name:    "multiple capabilities, 1 unknown, works",
```

Comment:
nit: Why "works"?  Although I guess you have `enabled capability works` in the previous entry, so maybe the pattern is just to end all test-case names with "works".